internal/layers/layers.go

```diff
@@ -146,6 +146,18 @@ func MountContainerLayers(ctx context.Context, layerFolders []string, guestRoot
 	hostPath := filepath.Join(layerFolders[len(layerFolders)-1], "sandbox.vhdx")
 	containerScratchPathInUVM := ospath.Join(uvm.OS(), guestRoot)
 	log.G(ctx).WithField("hostPath", hostPath).Debug("mounting scratch VHD")
+
```

Comment:
Sorry for being very nitpicky but I think it might be better to just add a new function with name `getScratchVhd` or something and then include the `filepath.Join(layerFolders[len(layerFolders)-1], "sandbox.vhdx")` part and EvalSymlinks part in that function so that we don't need to have the same EvalSymlinks code at multiple places.

Reply:
No it's perfectly valid, I'll add 👍

Reply:
Added